Meta/gn/secondary/Userland/Libraries/LibWeb/CookieStore/BUILD.gn

```diff
@@ -0,0 +1,5 @@
+source_set("CookieStore") {
```

Comment:
FYI you don't need to worry about the GN build. It's very, very broken at this point.
(You also don't need to undo this change, just a heads up for the future).